Contentful.AspNetCore/TagHelpers/ContentfulAssetTagHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Contentful.Core;
+using Contentful.Core.Images;
+using System.Threading.Tasks;
+
+namespace Contentful.AspNetCore.TagHelpers
+{
+    public class ContentfulImageTagHelper : TagHelper
+    {
+        private readonly IContentfulClient _client;
+
+        public ContentfulImageTagHelper(IContentfulClient client)
+        {
+            _client = client;
+        }
+
+        public string AssetId { get; set; }
+        public string Url { get; set; }
+        public int Width { get; set; }
+        public int Height { get; set; }
+        public ImageFormat Format { get; set; }
+        public ImageResizeBehaviour ResizeBehaviour { get; set; }
+        public ImageFocusArea FocusArea { get; set; }
+        public int? JpgQuality { get; set; }
+        public bool ProgressiveJpg { get; set; }
+        public int? CornerRadius { get; set; }
+        public string BackgroundColor { get; set; }
```

Comment:
Do you think this should be a specific type, which adds the `rgb:` to the color, or even create it from single values?

something like `Color(255,12,0x32)` which then translates to → `rgb:FF0C32`?